scripts/variantstore/wdl/extract/close_bed_file_gaps.py

```diff
@@ -0,0 +1,111 @@
+import csv
+import argparse
+import re
+import math
+
+# put these up here to get the defaults.  But override them later with user input
+binsize_kb = 100
+infile = f"gvs_vet_weights_{binsize_kb}kb.bed"
+outfile = f"gvs_vet_weights_{binsize_kb}kb_gaps_filled.bed"
+
+argParser = argparse.ArgumentParser()
```

Comment:
grumble grumble [Snake case](https://en.wikipedia.org/wiki/Snake_case) grumble grumble

Reply:
for the record, I hate using it in python scripts too.  When I'm copying someone's script I often keep their variables the same, but I can see that in this file I completely dropped snake case later.  I no longer feel bound to use those variables the way they were in the original github issue, and will make it something more pleasing for both of us to see lol

Reply:
Alright.  I cleaned up making the default that way anyhow, and changed everything (against my personal preferences) to be snake case.  THANKS MIGUEL